TransformationSystem/Client/Utilities.py

```diff
@@ -488,3 +490,7 @@ def isArchive( se ):
 def isFailover( se ):
   return DMSHelpers().isSEFailover( se )
 
+def getActiveSEs( seList, access = 'Write' ):
```

Comment:
There is already such a utility function in the DataManager class, where the default is Read. If it is useful at several place, then it should be made a general utility function